DapperDoodle.Tests/HostBuilderTests.cs

```diff
@@ -0,0 +1,43 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.Extensions.Hosting;
+using NUnit.Framework;
+
+namespace DapperDoodle.Tests
+{
+    public class HostBuilderTests
+    {
+        [Test]
+        public async Task AssertThatHostBuilderRegistersSuccessfully()
+        {
+            // Arrange
+            var expected = "HostBuilder was started successfully.";
+            
+            var hostBuilder = new HostBuilder()
+                .ConfigureWebHost(webhost =>
+            {
+                webhost.UseTestServer();
+                webhost.Configure(app => app
+                    .Run(async ctx => 
+                        await ctx
+                            .Response
+                            .WriteAsync(expected)));
+            });
+
+            // Act
+            var host = await hostBuilder.StartAsync();
+
+            var client = host.GetTestClient();
+            var response = await client.GetAsync("/");
+
+            response.EnsureSuccessStatusCode();
+            var actual = await response.Content.ReadAsStringAsync();
+            
+            // Assert
+            Assert.AreEqual(expected, actual);
```

Comment:
minor: consistency: suggest sticking to one assertions style throughout your project
minor: `Assert.{}` is deprecated in NUnit (not marked [Obsolete] because they don't want to introduce mad warnings in existing projects, but they won't support these assertions or add any more, so if you're going to use NUnit `Assert` assertions, rather stick to `Assert.That(...)`

Reply:
Thank you. I didn't know that. Also I wrote these tests before I included the NExpect package so I can also just switch to that.  🙂